### PR TITLE
Add setup script

### PR DESCRIPTION
**Pull request by coltborg**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+set -e
```
Comment (coltborg) [inline setup.sh:1]: @bencalegari I ran this and everything worked successfully! `brew update` took a long time, but I haven't ran it in awhile and that's somewhat expected.

Do you think we'd have another setup script in the starter app or would we point to this script in the starter app?

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+log_line '--- FormFlow Setup Script ---'
+
+if ! brew --version; then
+  curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh
+else
+  brew update
+fi
+
+log_line 'Installing brew packages'
+brew install openjdk@17 jenv gradle postgresql@14
```
Comment (coltborg) [inline setup.sh:24]: Do we need to install `openjdk@17` is it playing the same role as `temurin-17` in `jenv`? Or is it needed for something else?

File: setup.sh (hunk 1 above)
Comment (bencalegari) [reply setup.sh:1]: The latter seems sufficient for now. Glad it worked!

File: setup.sh (hunk 2 above)
Comment (bencalegari) [reply setup.sh:24]: Good catch, that's a mistake I accidentally carried over to the script.. I can just choose Temurin and change the package.

**Review by bencalegari — APPROVED** [review]

**Pull request merged**